ai21/clients/studio/resources/chat/base_chat_completions.py

```diff
@@ -40,6 +43,9 @@ def _create_body(
         stop: Optional[Union[str, List[str]]] | NotGiven,
         n: Optional[int] | NotGiven,
         stream: Literal[False] | Literal[True] | NotGiven,
+        tools: List[ToolDefinition] | NotGiven = NotGiven,
```

Comment:
Same as above. Here you're passing `type = type` and not `type = value`. Meaning -> `NotGiven = NOT_GIVEN` and not `NotGiven = NotGiven`

Reply:
Fix this across all areas you added please :)